boa/src/builtins/number/mod.rs

```diff
@@ -405,6 +411,132 @@ impl Number {
         Ok(Self::to_number(this))
     }
 
+    /// Builtin javascript 'parseInt(str, radix)' function.
+    ///
+    /// Parses the given string as an integer using the given radix as a base.
+    ///
+    /// An argument of type Number (i.e. Integer or Rational) is also accepted in place of string.
+    ///
+    /// The radix must be an integer in the range [2, 36] inclusive.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-parseint-string-radix
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
+    pub(crate) fn parse_int(
+        _this: &mut Value,
+        args: &[Value],
+        _ctx: &mut Interpreter,
+    ) -> ResultValue {
+        if args.len() > PARSE_INT_MAX_ARG_COUNT {
+            // Too many arguments.
+            return Err(Value::undefined());
+        }
```

Comment:
Where does the spec say this? If they are more arguments, I think we should just ignore them, as other interpreters do.

Reply:
Addressed in latest commit